app/controllers/registrations_controller.rb

```diff
@@ -6,15 +6,18 @@ def new
             render 'choice'
         else
             @role = params[:role] || 'fan'
-            super
+            build_resource
+            #yield resource if block_given?
+            #respond_with resource
+
```

Comment:
comments

Reply:
@ESteinman Can you clean this up for me?